sdk/communication/Azure.Communication.Identity/api/Azure.Communication.Identity.netstandard2.0.cs

```diff
@@ -45,3 +47,19 @@ public enum ServiceVersion
         public override string ToString() { throw null; }
     }
 }
+namespace Azure.Communication.Identity.Models
+{
+    public partial class CommunicationTurnCredentialsResponse
+    {
+        internal CommunicationTurnCredentialsResponse() { }
+        public System.DateTimeOffset ExpiresOn { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<Azure.Communication.Identity.Models.CommunicationTurnServer> TurnServers { get { throw null; } }
+    }
+    public partial class CommunicationTurnServer
+    {
+        internal CommunicationTurnServer() { }
+        public string Credential { get { throw null; } }
+        public string Urls { get { throw null; } }
```

Comment:
Does this contain multiple urls?

Reply:
It doesn't. I had discussed this with Andrew (@ajpeacock0 ) about this few days ago. It looks like according to the [WebRTC standard, `url` is deprecated.](https://developer.mozilla.org/en-US/docs/Web/API/RTCIceServer#obsolete_properties)

Reply:
Oh nice, it is good that we are following the standards. One observation thought, [urls](https://developer.mozilla.org/en-US/docs/Web/API/RTCIceServer/urls) could be an array as well. 
```
urls = iceServerUrl | [ url1, ..., urlN ]
```
which unfortunately not easy to do in swagger. But If we switch to `array` at least we will be future proof

Reply:
Will follow up offline with you and Andrew.

Reply:
Talked to Reza and Andrew offline, will address this in a subsequent PR as this needs to be coordinated with both swagger and service side updates.